### PR TITLE
new formating

### PR DESCRIPTION
**Pull request by perezdf**

### CODE REVIEW

File: spring-boot/admin/src/main/java/eu/coatrack/admin/WebConfig.java [hunk 1]
```diff
@@ -59,13 +60,8 @@ public void addInterceptors(InterceptorRegistry registry) {
...
         // Simple strategy: only path extension is taken into account
-        configurer.favorPathExtension(true).
-                favorParameter(false).
-                parameterName("mediaType").
-                ignoreAcceptHeader(false).
-                useJaf(false).
-                defaultContentType(MediaType.APPLICATION_JSON).
-                mediaType("xml", MediaType.APPLICATION_XML).
-                mediaType("json", MediaType.APPLICATION_JSON);
+        configurer.favorPathExtension(true).favorParameter(false).parameterName("mediaType").ignoreAcceptHeader(false)
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/main/java/eu/coatrack/admin/WebConfig.java:63]: This re-formatting makes the code harder to read, as the newlines were added on purpose. Is it possible to configure the formatter to "not join lines".

File: spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminApiKeysController.java [hunk 2]
```diff
@@ -256,18 +252,16 @@ public ApiKey getByIdRest(@PathVariable("id") long id) throws IOException {
...
         for (ApiKey apiKey : apiKeys) {
             String apiKeyString = apiKey.getKeyValue();
-            List<String> proxiesUrlList = proxyRepository.customSearchForAllProxiesForGivenServiceApiId(apiKey.getServiceApi().getId())
-                    .stream()
-                    .filter(proxy -> proxy.getPublicUrl() != null)
-                    .filter(proxy -> proxy.getPublicUrl() != "")
-                    .map(Proxy::getPublicUrl)
-                    .collect(Collectors.toList());
+            List<String> proxiesUrlList = proxyRepository
+                    .customSearchForAllProxiesForGivenServiceApiId(apiKey.getServiceApi().getId()).stream()
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminApiKeysController.java:262]: This is another example: the newlines were added on purpose to make it easy to read/understand. If possible, the formatter should not join lines here.

**Review by gr-hovest-atb — CHANGES_REQUESTED** [review]

File: spring-boot/admin/src/main/java/eu/coatrack/admin/WebConfig.java (hunk 1 above)
Comment (perezdf) [reply spring-boot/admin/src/main/java/eu/coatrack/admin/WebConfig.java:63]: take into account the line is "join" in the unify form, you can always clone and format in your most preferable IDE and formating, do whatever you want and commit which will reformat joining the lines

Comment (gr-hovest-atb) [reply spring-boot/admin/src/main/java/eu/coatrack/admin/WebConfig.java:63]: In general I agree: everyone can use their own IDE formatting and then the maven plugin will unify the formatting. That approach sounds fine.

The only problem I see: sometimes it makes sense to manually add newlines to make the code easier to read, which also helps reviewers during pull request reviews. But these newlines get lost during the auto-formatting. Therefore I would suggest a slightly changed config, which contains the setting: `<setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="false"/>` (by default this is `true`).

See my pull request: #143

**Pull request closed without merging**